content/tutorials/spatial_wfs_services/index.Rmd

```diff
@@ -158,6 +158,12 @@ bwk_client$
   findFeatureTypeByName("BWK:Bwkhab")$
   getDescription() %>%
   map_chr(function(x){x$getName()})
+
+# or shorter
+bwk_client$
+  describeFeatureType(typeName = "BWK:Bwkhab") %>%
+  map_chr(function(x){x$getName()})
+  
```

Comment:
When rendering the whole chunk, the result will be printed twice. Intended as such?

Reply:
I think it is OK to explicitly show they are the same